Commands/ExportCommand.php

```diff
@@ -232,6 +240,17 @@ private function prepareExportInputValidation(InputInterface $input): void
                 $this->productStreamId = $this->getProductStreamIdByName($productStream);
             }
         }
+        
+        $orderstate = $input->getOption('orderstate');
+        if ($orderstate !== null) {
+            if (is_numeric($orderstate)) {
+                $this->orderstate = (int) $orderstate;
+            } elseif ($orderstate == 'cancelled') {
```

Comment:
why do we need a special case for "Cancelled" ?

Reply:
because the value "-1" doesn't work ;-)
for example, we needed this case for our customer.

Reply:
okay, we will have a look :+1: